### PR TITLE
Added  Presentation  Section #19

### PR DESCRIPTION
**Pull request by itsashishbharti**
mp
mpresentation

### CODE REVIEW

File: styles/presentation.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+        #presentation > h2
+        {
+	   color: gray;
```
Comment (amezng) [inline styles/presentation.css:4]: Use css variable

File: styles/presentation.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            gap: 40px;
+            width: 1000px;
+            margin: 0 auto;
+        }
+        #presentation 
+        {
+            margin: 50px 0;
+        }
+        #presentation h4
+        {
```
Comment (amezng) [inline styles/presentation.css:20]: For h4 use primary color

**Review by amezng — CHANGES_REQUESTED** [review]

File: styles/presentation.css (hunk 2 above)
Comment (itsashishbharti) [reply styles/presentation.css:20]: issue resolved

File: styles/presentation.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
+
+        #presentation > h2
+        {
+	   color: var(--secondary-font-color);
```
Comment (amezng) [inline styles/presentation.css:4]: Use primary

**Review by amezng — CHANGES_REQUESTED** [review]

**Pull request merged**